docs/development.md

```diff
@@ -1,10 +1,11 @@
 # Running the SHIELD Dashboard locally
 
-1. `echo DEBUG=True >> .env`
+1. `echo DEBUG=true >> .env`
```

Comment:
Can you say more about this change? I notice the [python-decouple documentation](https://pypi.python.org/pypi/python-decouple#env-file) uses the upper-case *T*.